Define and activate KVM 11.2.2 release (#219)
giantswarm/releases: release-notes/kvm/v11.2.2.md

## :zap: Giant Swarm Release 11.2.2 for KVM :zap:

**If you are upgrading from 11.2.1, upgrading to this release will not roll your nodes. It will only update the apps.**

> This release includes couple reliability improvements to pre-installed apps such as NGINX Ingress Controller and kube-state-metrics.

Author: ```suggestion
The release includes reliability improvements to these pre-installed apps: NGINX Ingress Controller and kube-state-metrics.
```
